src/testdrive/src/action/sql.rs

```diff
@@ -136,18 +136,23 @@ pub async fn run_sql(mut cmd: SqlCommand, state: &mut State) -> Result<ControlFl
         | Statement::GrantRole { .. }
         | Statement::RevokePrivileges { .. }
         | Statement::RevokeRole { .. } => {
+            let inconsistencies = reqwest::get(&format!(
+                "http://{}/api/coordinator/check",
+                state.materialize_internal_http_addr,
+            ))
+            .await?
+            .text()
```

Comment:
You should be able to call `.json()` here which will parse the body as JSON

Reply:
Ah thanks for that. I was thinking about leaving it as is b/c i've added more context when each step can fail which has been helpful while debugging